docs/kyma/docs/017-details-local-installation.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Local installation
+type: Details
+---
+
+For better understanding of complex installation process, this guide describes it step by step.
+
+## Start local installation
+
+To fire local installation run following command:
+```
+./installation/cmd/run.sh
+```
+
+This script sets up default parameters, starts Minikube, builds Kyma-Installer, generates local configuration, creates `installer` Custom Resource and sets up Installer. Subsequent sections provide a detailed description of each step.
+
+### Installation parameters
+
+You can execute the `installation/cmd/run.sh` script with the following parameters:
+
+- `--skip-minikube-start` - it skips execution of the `installation/scripts/minikube.sh` script. See the "Start Minikube" section for more details.
+- `--vm-driver` -  either `virtualbox` or `hiperkit` depending on your operating system
+
+## Start Minikube
+
+> **NOTE:** To work with Kyma, use only the provided scripts and commands. Kyma does not work on a basic Minikube cluster that you can start using the `minikube start` command or stop with the `minikube stop` command. If you don't need Kyma on Minikube anymore, remove the cluster with the `minikube delete` command.
```

Comment:
```suggestion
> **NOTE:** To work with Kyma, use only the provided scripts and commands. Kyma does not work on a default Minikube cluster started with `minikube start` command. If you don't need Kyma on Minikube anymore, remove the cluster with the `minikube delete` command.
```

Reply:
this one is note copied from docs/kyma/docs/031-gs local-installation.md